releasenotes/notes/sni-dnat-default.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: release-notes/v2
+kind: feature
+area: networking
+issue:
+- 27749
+releaseNotes:
+- |
+  **Enhanced** the default installation of gateways to not configure clusters for `AUTO_PASSTHROUGH`, reducing memory costs.
```

Comment:
may be state it as updated rather than enhanced ?

Reply:
good idea, done